src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -369,6 +373,24 @@ private function getProgressBar(): ProgressBar
             ?? throw new RuntimeException('The ProgressBar is not started.');
     }
 
+    /**
+     * @param iterable<string, iterable|string|TreeNode> $nodes
```

Comment:
Some words or example about what can be passed would be nice either here or in TreeHelper or both

Reply:
HA ! It is to avoid back and forth (with a future link to the doc) :), a sample like :

```php

    $treeHelper = new TreeHelper($output, TreeNode::fromValues([
	    'a' => [
	        'a1',
	        'a2'
	    ],
	    'b',
	    'c' => [
	        'c1',
	        'c2' => [
	            'c21' => [
	                'c211',
	                'c212',
	            ],
	        ],
	    ],
		],
        new TreeNode($rootNode),
    ), TreeStyle::rounded());

        $treeHelper->render();

```